Frends.AzureBlobStorage.ListBlobsInContainer/Frends.AzureBlobStorage.ListBlobsInContainer/ListBlobsInContainer.cs

```diff
@@ -0,0 +1,151 @@
+﻿using System;
+using System.ComponentModel;
+using Azure.Storage.Blobs;
+using Azure;
+using Frends.AzureBlobStorage.ListBlobsInContainer.Definitions;
+using System.Threading.Tasks;
+using Azure.Storage.Blobs.Models;
+using System.Collections.Generic;
+#pragma warning disable CS1573
+
+namespace Frends.AzureBlobStorage.ListBlobsInContainer
+{
+    public class AzureBlobStorage
+    {
+        /// <summary>
+        ///     List of blobs in container.
+        ///     [Documentation](https://tasks.frends.com/tasks#frends-tasks/Frends.AzureBlobStorage.ListBlobsInContainer)
+        /// </summary>
+        /// <param name="source">Source values.</param>
+        /// <returns>List of blobs in container with chosen listing structure.</returns>
+        /// <exception cref="Exception">Authentication exception.</exception>
+
+        public static async Task<Result> ListBlobsInContainer([PropertyTab] Source source)
+        {
+            BlobContainerClient blobContainerClient = null;
+            var blobsList = new List<Result>();
+            var authSas = false;
+
+            switch (source.AuthenticationMethod)
+            {
+                case AuthenticationMethod.Connectionstring:
+                    if (string.IsNullOrEmpty(source.ConnectionString))
+                        throw new Exception("Connection string required.");
+                    blobContainerClient = new BlobContainerClient(source.ConnectionString, source.ContainerName);
+                    break;
+
+                case AuthenticationMethod.Sastoken:
+                    if (string.IsNullOrEmpty(source.SasToken) || string.IsNullOrEmpty(source.Uri))
+                        throw new Exception("SAS Token and URI required.");
+                    authSas = true;
+                    blobContainerClient = new BlobContainerClient(new Uri($"{source.Uri}/{source.ContainerName}?"), new AzureSasCredential(source.SasToken));
+                    break;
+            }
+
+            if (source.FlatBlobListing)
+            {
+                var enumerable = blobContainerClient.GetBlobsAsync(BlobTraits.None, BlobStates.None, string.IsNullOrWhiteSpace(source.Prefix) ? null : source.Prefix).AsPages();
```

Comment:
This method is an asynchronous method, so you should pass cancellation token to the method. Also, you could move the `string.IsNullOrWhiteSpace(source.Prefix) ? null : source.Prefix`-check to it's own line above this line and save the result of the check to a variable to make this method call look a bit more simpler.